spec/stats/kernel-selftests/net-14

```diff
@@ -0,0 +1,89 @@
+# selftests: net: udpgro_fwd.sh
+# IPv4
+# No GRO
+# ./udpgro_fwd.sh: line 135: [: !=: unary operator expected
+# ./udpgro_fwd.sh: line 145: [: too many arguments
+#  ok
+# GRO frag list
+# ./udpgro_fwd.sh: line 135: [: !=: unary operator expected
+# ./udpgro_fwd.sh: line 145: [: too many arguments
+#  ok
+# GRO fwd
+# ./udpgro_fwd.sh: line 135: [: !=: unary operator expected
+# ./udpgro_fwd.sh: line 145: [: too many arguments
+#  ok
+# UDP fwd perf                            udp rx:      4 MB/s     3536 calls/s
```

Comment:
this part is same as net-13, it doesn't help to discover new problem, help reduce unnecessary info.